app/demo/cmd/main.go

```diff
@@ -17,7 +17,7 @@ func main() {
 	// https://pkg.go.dev/github.com/spiffe/spike-sdk-go/api#Close
 	defer api.Close() // Close the connection when done
 
-	path := "/tenants/demo/db/creds"
+	path := "tenants/demo/db/creds"
```

Comment:
Vault, OpenBao, and other similar key-value secret stores prefer the keys to not have a trailing slash (but they don't strictly restrict it either) -- changed the example to follow the suit.